api/service/resharding/service.go

```diff
@@ -49,6 +58,18 @@ func (s *Service) Run(stopChan chan struct{}, stoppedChan chan struct{}) {
 
 // DoService does role conversion.
 func (s *Service) DoService() {
+	tick := time.NewTicker(ReshardingCheckTime)
```

Comment:
Resharding is block number based instead of time based. Do we really need a service?

Reply:
Regarding my PR, the resharding service keeps checking the new  status from beacon blockchain - currently i designs it checking every single second.

Is there any suggestionof not having a service?

Reply:
FYI, this is still an ongoing work